README.md

```diff
@@ -1,31 +1,33 @@
-# Hypershift POC
+## HyperShift
 
-All the following assumes `KUBECONFIG` points to the management cluster.
+Guest clustering for [OpenShift](https://openshift.io).
 
-Build binaries: 
+### Prerequisites
 
-```
-$ make
-```
+* [Kustomize](https://kustomize.io)
+* Admin access to an OpenShift cluster.
+* The `oc` and `kubectl` CLI tools.
```

Comment:
Is there something we get from kubectl that we can't do with oc?

Reply:
Replaced all references to `kubectl` with `oc`